x-pack/plugins/infra/public/components/asset_details/tabs/overview/alerts.tsx

```diff
@@ -106,12 +124,21 @@ const MemoAlertSummaryWidget = React.memo(
       baseTheme: charts.theme.useChartsBaseTheme(),
     };
 
+    const getAlertsCount = (alertsCount: AlertsCount) => {
+      const hasActiveAlerts =
+        typeof alertsCount?.activeAlertCount === 'number' && alertsCount?.activeAlertCount > 0;
+
+      setIsAlertSectionOpen(hasActiveAlerts ? 'open' : 'closed');
+      setActiveAlertsCount(alertsCount?.activeAlertCount ?? 0);
+    };
+
     return (
       <AlertSummaryWidget
         chartProps={chartProps}
         featureIds={infraAlertFeatureIds}
         filter={alertsQuery}
         timeRange={summaryTimeRange}
+        getAlertsCount={(alertsCount) => getAlertsCount(alertsCount)}
```

Comment:
Perhaps we could use the prefix `on` here since it's an event - it's usually the convention for events in js - wdyt? eg: `onAlertsCountRetrieved`

Reply:
Changed to `onLoaded` but thanks, makes sense 👍